lib/pluginmanager/bundler/logstash_injector.rb

```diff
@@ -93,6 +93,9 @@ def inject(gemfile_path, lockfile_path, dependencies)
 
           builder.eval_gemfile("bundler file", gemfile.generate())
           definition = builder.to_definition(lockfile_path, {})
+          LogStash::Bundler.specific_platforms(definition.platforms).each do |specific_platform|
```

Comment:
nit: this looks a lot like `LogStash::Bundler::genericize_platform`, except it doesn't ensure that we have a generic java in the list.

Is there a danger in not invoking something like `definition.add_platform("java")`?

Reply:
I've added something the functional equivalent to this - it appears not to matter if `java` is already in the list of platforms, so there doesn't appear to be a danger.